forge-gui/res/cardsfolder/upcoming/summoners_grimoire.txt

```diff
@@ -0,0 +1,13 @@
+Name:Summoner's Grimoire
+ManaCost:3 G
+Types:Artifact Equipment
+K:Job select
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddType$ Shaman | AddTrigger$ TrigAttack | Description$ Equipped creature is a Shaman in addition to its other types and has "Whenever this creature attacks, you may put a creature card from your hand onto the battlefield. If that card is an enchantment card, it enters tapped and attacking."
+SVar:TrigAttack:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigChoose | TriggerZones$ Battlefield | OptionalDecider$ You | TriggerDescription$ Whenever this creature attacks, you may put a creature card from your hand onto the battlefield. If that card is an enchantment card, it enters tapped and attacking.
+SVar:TrigChoose:DB$ ChooseCard | Mandatory$ True | ChoiceZone$ Hand | Choices$ Creature.YouOwn | ChoiceTitle$ Choose a creature card in your hand | SubAbility$ DBChangeOne
```

Comment:
@tool4ever I wish that would work without extra Choose Ability 😒

Reply:
doesn't look like other cards with such condition exist yet, otherwise some extra code could be worth it

Reply:
Took a small peak, 
Might change that in the future 

Good for now I think 🤔 

Question for later what happens if the creature would only be an Enchantment on the battlefield, but I can ask the Rules Manager later